go/chat/commands/msg.go

```diff
@@ -26,6 +26,12 @@ func (d *Msg) Execute(ctx context.Context, uid gregor1.UID, convID chat1.Convers
 	if !d.Match(ctx, text) {
 		return ErrInvalidCommand
 	}
+	defer func() {
+		if err != nil {
+			d.getChatUI().ChatCommandStatus(ctx, convID, "Failed to send message",
```

Comment:
headline can fail if the length is too long as well, can you add a status there or a ticket to address other commands?